Misc/NEWS.d/next/Core and Builtins/2024-02-03-04-07-18.gh-issue-114887.uLSFmN.rst

```diff
@@ -0,0 +1 @@
+Changed socket type validation in loop.create_datagram_endpoint() to accept all non-stream sockets. This fixes a regression in compatibility with raw sockets.
```

Comment:
```suggestion
Changed socket type validation in :meth:`loop.create_datagram_endpoint` to accept all non-stream sockets.
This fixes a regression in compatibility with raw sockets.
```

Reply:
hmm it doesn't work.. :(

Reply:
Okay it works

```suggestion
Changed socket type validation in :meth:`~asyncio.loop.create_datagram_endpoint` to accept all non-stream sockets.
This fixes a regression in compatibility with raw sockets.
```

Reply:
<img width="921" alt="스크린샷 2024-02-03 오후 6 13 17" src="https://github.com/python/cpython/assets/5110323/34da7850-126c-4338-819b-da7bcc735b19">